samples/bluetooth/mesh/ble_peripheral_lbs_coex/sample.yaml

```diff
@@ -25,3 +25,15 @@ tests:
       - bluetooth
       - ci_build
       - sysbuild
+  sample.bluetooth.mesh.ble_and_mesh.ext_flash_settings:
+    sysbuild: true
+    build_only: true
+    extra_args:
+      - FILE_SUFFIX=ext_flash
```

Comment:
Did not get why FILE_SUFFIX is set here if it isn't used anywhere? There are no even files for file suffix either.
(comment is relevant for any sample change in PR)